addons/web/static/src/views/basic_relational_model.js

```diff
@@ -566,6 +566,8 @@ export class Record extends DataPoint {
      * @param {boolean} [options.useSaveErrorDialog=false] displays a custom
      *  dialog and await the response from this dialog when an error is
      *  returned by the server.
+     * @param {boolean} [options.throwOnError=false] throws the saving error if
```

Comment:
I don't really have a strong opinion on this, seems a bit niche but also does not hurt to have more options.

Will ask the framework team when we get close to being done with the rest :)